src/api/route-services/pipeline-response.js

```diff
@@ -88,18 +88,21 @@ const pipelineResponse = async (io, message) => {
   }
 
   const statusRes = await getPipelineStatus(experimentId, constants.QC_PROCESS_NAME);
+
+  const statusResToSend = { pipeline: statusRes[constants.QC_PROCESS_NAME] };
+
   pipelineHook.run(taskName, {
     experimentId,
     output,
-    statusRes,
+    statusResToSend,
```

Comment:
@cosa65 I think that the hook is expecting a `statusRes`, not a `statusResToSend`. See your own changes updating `PipelineRunETag`

Reply:
You're right! I changed this and forgot to keep the same name. Thanks for the catch.

Reply:
Added a commit for that